[7.0] Update & Simplify install instructions

andrerom opened the pull request:
Now that we require PHP 7.1 and have greatly improved performance of backend placing hinst on performance tuning does not need to take space here anymore.

Besides that:
- updated to use dev, with symfony 3 builtin server is only available in dev by default
- update to use `/admin` url for backend
- somewhat simplified the text a bit

Open question:
- [x] Do we need to run the assetic:dump command? As it's already ran by composer create-project which is what has been done when this is shown, **ok if we remove?**

alongosz commented on eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php:
```diff
@@ -119,24 +119,18 @@ public static function installWelcomeText(Event $event)
 - "ezplatform:install" has different installer <type>s depending on your install, see <fg=green>INSTALL.md</> or <fg=green>README.md</> for which one to use.
-- For development use you can enable full debugging by setting SYMFONY_ENV to "dev".
-- Last command will give you url to frontend of installation, add "/ez" to reach backend.
+- Instructions above assumes the CLI user you execute commands with is same that extracted/installed the software.
```
Hmm, something here doesn't sound right for me. Maybe:
`The instructions above assume the CLI user you execute commands with is the same that extracted/installed the software.`
Not an expert, I think it's safer to ping @DominikaK :grinning:

andrerom replied:
wdyt @DominikaK ? :)

DominikaK replied:
Agree with @alongosz, maybe even `is the same one that`...

andrerom replied:
@DominikaK maybe add `these` to? `The instructions above assume the CLI user you execute these commands with is the same one that extracted/installed the software.` ?

DominikaK replied:
Even better :)

andrerom replied:
Fixed in 492847b

SylvainGuittard commented on eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php:
```diff
@@ -119,24 +119,18 @@ public static function installWelcomeText(Event $event)
+<comment>    $  export SYMFONY_ENV="dev"</comment>
 <comment>    $  php ${consoleDir}/console ezplatform:install <type></comment>
 <comment>    $  php ${consoleDir}/console assetic:dump</comment>
 <comment>    $  php ${consoleDir}/console server:run</comment>
```
You can only execute this command if you have `symfony/web-server-bundle` in `composer.json`. And this is not the case in this repo or eZ Platform meta repo.

andrerom replied:
@SylvainGuittard it is there in 2.0: https://github.com/ezsystems/ezplatform/blob/2.0/app/AppKernel.php#L56

SylvainGuittard replied:
But not in `composer.json`: https://github.com/ezsystems/ezplatform/blob/2.0/composer.json#L54

alongosz replied:
@SylvainGuittard actually... it is :grinning: 
```bash
$ composer why symfony/web-server-bundle
symfony/symfony  v3.4.0-RC2  replaces  symfony/web-server-bundle (self.version)
```

SylvainGuittard replied:
That's very good. I did not know about that. Thanks @alongosz

andrerom commented:
Ok, fixed comments and simplified a bit more. I'm assuming the assetic:dump was a copy paste from install doc which had it for when people install from archive and need to setup symlinks again. 

review ping @alongosz @DominikaK @SylvainGuittard

DominikaK commented on eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php:
```diff
@@ -119,24 +119,18 @@ public static function installWelcomeText(Event $event)
+- "ezplatform:install" has different installer <type>s depending on your install, run command with <comment>--help</comment> to see your options.
+- Instructions above assume the CLI user you execute these commands with is the same one that extracted/installed the software,
+  otherwise see install instructions for directory permissions.
+- Last command will give you url to frontend of installation, add "/admin" to reach backend.
```
`- The last command will give you the url to the frontend of the installation, add "/admin" to reach backend.`

andrerom replied:
Thanks, fixed in f87e2d7
